packages/asc-ui/src/components/Downloader/Downloader.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import { Download } from '@datapunt/asc-assets'
+import DownloaderStyle from './DownloaderStyle'
+import { Row, Column } from '../Grid'
+import Button from '../Button'
+import Image from '../Image'
+import DownloaderContent from './DownloaderContent'
+import Typography from '../Typography'
+import Icon from '../Icon'
+
+interface Props {
+  imageSrc: string
+  onClick: (event: React.MouseEvent<HTMLButtonElement>) => void
+  description: string
+}
+
+const Downloader: React.FC<Props> = ({
```

Comment:
"Downloader" might sound like a download module, while it's actually just an image of a (pdf) file, with a button to download a file, right?. Consider renaming it to sth like "DocumentPreview" or "DocumentCover", something like that?

Reply:
Renamed to DocumentCover